azure_functions_worker/dispatcher.py

```diff
@@ -90,6 +87,8 @@ async def connect(cls, host: str, port: int, worker_id: str,
         disp._grpc_thread.start()
         await disp._grpc_connected_fut
         logger.info('Successfully opened gRPC channel to %s:%s', host, port)
+        logger.info('Python worker sync threadpool max workers: %s',
```

Comment:
If possible, can we merge the two log statements into one? :) Trying to reduce logging that happens from the Python worker side.